<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContextSidebar</class>
 <widget class="QWidget" name="ContextSidebar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>490</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ContextSidebar {
	border-right: 1px solid #34343426;
	border-top-left-radius: 16px;
	border-top-right-radius: 0px;
}

QPushButton {
	text-align: left;
    background: transparent;
    border: none;
    font-size: 14px;
    font-weight: 600;
    color: #22084B;
    padding: 0px 10px;
}

QPushButton:hover,
QPushButton:pressed,
QPushButton:checked {
    background-color: #EEF4F6;
    border-radius: 8px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>22</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>22</number>
   </property>
   <item>
    <widget class="QPushButton" name="firstButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>44</height>
      </size>
     </property>
     <property name="text">
      <string>Basic Settings</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/images/ProjectSettings/basic_settings.svg</normaloff>:/images/ProjectSettings/basic_settings.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="secondButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>44</height>
      </size>
     </property>
     <property name="text">
      <string>Integrations</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/images/ProjectSettings/integrations.svg</normaloff>:/images/ProjectSettings/integrations.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="thirdButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>44</height>
      </size>
     </property>
     <property name="text">
      <string>AI Settings</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/images/ProjectSettings/ai_settings.svg</normaloff>:/images/ProjectSettings/ai_settings.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="fourthButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>44</height>
      </size>
     </property>
     <property name="text">
      <string>User Management</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/images/ProjectSettings/user_management.svg</normaloff>:/images/ProjectSettings/user_management.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
